tests/integration/it_locations_test.py

```diff
@@ -37,16 +37,33 @@ def test_create(client):
 
 
 def test_edit(client):
-    assert client.get("/locations/edit/1").status_code == HTTPStatus.OK
+    location_old = factories.LocationFactory()
+    location_data = {
+        "name": "Name",
+        "code": "Code",
+        "country": "Country",
+        "region": "Region",
+        "city": "City",
+        "address": "Address",
+        "longitude": "0.0000000",
+        "latitude": "0.0000000",
+        "type": "Type",
+        "status": "Active",
+        "comment": "No comments",
+    }
+    url = url_for("location.edit", id=location_old.id)
```

Comment:
@vladarefiev one time variable here, do you really like it like this? At least can we name it ``edit_url``? :D and there is one more bellow

Reply:
@amihaiemil done